migration/keeper/keeper-migration.py

```diff
@@ -233,16 +235,22 @@ def get_keeper_records(params: KeeperParams) -> list:
     return records
 
 
-
 def get_keeper_folders(params: KeeperParams) -> dict:
     """Fetches all folders from Keeper (subfolders + shared folders)."""
     folders = {}
 
     if params.folder_cache:
-        folders.update({uid: folder["name"] for uid, folder in params.folder_cache.items()})
+        folders.update(
```

Comment:
This lambda function wasn't correctly specified, but this works when running this against a test account.